client/src/components/visualisation/ConfigMenu.jsx

```diff
@@ -0,0 +1,253 @@
+import React, { PropTypes } from 'react';
+import DashSelect from '../common/DashSelect';
+import ColumnMenu from './configMenu/ColumnMenu';
+import LabelColumnMenu from './configMenu/LabelColumnMenu';
+import LabelInput from './configMenu/LabelInput';
+
+
+const getDatasetArray = datasetObject => {
+  const datasetArray = [];
+  const sortFunction = (a, b) => a.name.toLowerCase() > b.name.toLowerCase();
+
+  Object.keys(datasetObject).forEach(key => {
+    datasetArray.push(datasetObject[key]);
+  });
+
+  datasetArray.sort(sortFunction);
+
+  return datasetArray;
+};
+
+const getDatasetOptions = datasetArray => {
+  const output = [];
+
+  datasetArray.forEach(option => {
+    output.push({
+      value: option.id, label: option.name,
+    });
+  });
+
+  return output;
+};
+
+const getDashSelectOptionsFromColumnArray = array => {
+  const output = [];
+
+  if (array) {
+    array.forEach((entry, index) => {
+      output.push({
+        value: index, label: entry.title,
+      });
+    });
+  }
+  return output;
+};
+
+const getSubTitle = subtitle => (
+  <h3 className="subtitle">{subtitle}</h3>
+);
```

Comment:
I wonder if it would be better to turn this into a component, something like 
```jsx
const SubTitle = ({ children }) => (
  <h3 className="subtitle">{children}</h3>
);

SubTitle.propTypes = {
  children: PropTypes.node.isRequired,
};
```
so the call-sites can be written as `<SubTitle>X-Axis</SubTitle>` instead of `{getSubTitle('X-Axis')}`?

Note that "functional" components doesn't have to be in its own file so we can still treat it as just a helper component.